.github/workflows/workflow.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       run: |
         echo "::set-output name=dir::$(npm config get cache)"
 
-    - uses: actions/cache@v1
+    - uses: ./
```

Comment:
awesome.  PRs will now test what's changing with the resultant merge into master :)